lib/commands/general.js

```diff
@@ -114,6 +114,16 @@ commands.getWindowSize = async function (windowHandle = 'current') {
   }
 };
 
+// For W3C
+commands.getWindowRect = async function () {
+  const rect = await this.getWindowSize();
+  if (_.isObject(rect)) {
```

Comment:
-> isPlainObject

Reply:
btw, why do we need this condition?

Reply:
Because when I run test code https://github.com/appium/appium-xcuitest-driver/pull/607/files#diff-31e3d78d9e422a2511e656a74368f4d9R97 , I encountered `undefined` error here. (In this case, no mocking is the reason though.)

So, I thought we should care the case in JS world. When I read code in `getWindowSize `, it looks raising exceptions like the following and I also thought we don't need to care `undefined` case. But I have no confidence in it in JS world.
Thus, I  put the condition flow here.

```
  if (res && res.status && parseInt(res.status, 10) !== 0) {
    throw errorFromCode(res.status, res.value);
  }
```